alarms.tf

```diff
@@ -0,0 +1,43 @@
+module "ddb_consume_read_capacity_alarms" {
+  source              = "terraform-aws-modules/cloudwatch/aws//modules/metric-alarm"
```

Comment:
Terraform AWS modules don't have to have dependencies on other modules.

Users will have to create their own alarms outside of the module.

If you can put a small example into README and in example we can have it but there is no need to touch any other files.

Reply:
What do you mean "Users will have to create their own alarms outside of the module." ? my idea with this PR is to optionally create alarms. If you don't agree fell free to decline this PR.

Reply:
I mean that the users will have to call several modules to compose their infrastructure - one module for DynamoDB table, another for CloudWatch metrics.

There is nothing we can do in the module other than updating README and/or example code to show how to do this.